Don't input text when HELP_TEXT is rendered

Apaczer opened the pull request:
it can lead to crashing often
+ close help_view also with F1

Apaczer committed:
don't pass inputs when `help_view` is active

rofl0r commented:
thanks, i fixed the crashing via a assert->return change in ezsdhl.h, and solved the "what kind of input is expected" in a more generic manner. apart from that, i don't like the bool type, it reminds me of C++ or Java...

Apaczer commented:
never touched Java, but always though that `bool` is more of a plain C construct 😄.

> i fixed the crashing via a assert->return

Thanks, it is much better that way - this also fixed other assertion error on smaller screen sizes! 

> and solved the "what kind of input is expected"

This program will be used mainly on SBC device with only a few buttons (gaming handheld), and it was a nice touch to put numbers with modifier keys on "go to page n", maybe some UP/DOWN reading could be added to increase/decrease outputted page_n value.
